packages/mobile/src/recipients/RecipientPicker.tsx

```diff
@@ -181,6 +215,32 @@ function RecipientPicker(props: RecipientProps) {
     }
   }
 
+  const mapResolutionToRecipient = (resolution: NameResolution) => {
+    const lowerCaseAddress = resolution.address.toLowerCase()
+    switch (resolution.kind) {
+      case ResolutionKind.ADDRESS:
+        return getRecipientFromAddress(lowerCaseAddress, recipientInfo)
+      case ResolutionKind.NOM:
+        return {
+          address: lowerCaseAddress,
+          name: t('nomSpaceRecipient', { name: resolution.name ?? props.searchQuery }),
+        }
+      default:
+        return getRecipientFromAddress(lowerCaseAddress, recipientInfo)
+    }
+  }
+
+  const buildSections = (defaultSections: Section[]) => {
+    if (resolveKitCall.result && resolveKitCall.result.resolutions.length > 0) {
```

Comment:
Do we want this new feature to be behind a feature flag?

Reply:
My opinion is no, if someone doesn't look for a nom name then they just won't see anything, but don't think we'll learn anything from people having the search not work if they do search for one.